tree/dataframe/src/RArrowDS.cxx

```diff
@@ -241,46 +249,46 @@ class TValueGetter {
 
    // Convenience method to avoid code duplication between
    // SetEntry and InitSlot
-   void UncachedSlotLookup(unsigned int slot, ULong64_t entry)
+   arrow::Status UncachedSlotLookup(unsigned int slot, ULong64_t entry)
    {
       // If entry is greater than the previous one,
       // we can skip all the chunks before the last one we
       // queried.
       size_t ci = 0;
-      assert(slot < fLastChunkPerSlot.size());
-      if (fLastEntryPerSlot[slot] < entry) {
-         ci = fLastChunkPerSlot.at(slot);
+      // assert(slot < fLastChunkPerSlot.size());
```

Comment:
Why leaving the asserts here, but commented? (Why not just leave them?)

Reply:
Added back.